CHANGELOG.md

```diff
@@ -2,6 +2,47 @@
 
 ## Unreleased
 
+## v0.32.0
+
+# 🎉 OpenTelemetry Collector Contrib v0.32.0 (Beta) 🎉
+
+The OpenTelemetry Collector Contrib contains everything in the [opentelemetry-collector release](https://github.com/open-telemetry/opentelemetry-collector/releases/tag/v0.32.0) (be sure to check the release notes here as well!). Check out the [Getting Started Guide](https://opentelemetry.io/docs/collector/getting-started/) for deployment and configuration information.
+
+## 🚀 New components 🚀
+
+- [`cumulativetodelta` processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/cumulativetodeltaprocessor) to convert cumulative sum metrics to cumulative delta
+
+## 💡 Enhancements 💡
+
+- `logzio` exporter:
+  - New implementation of an in-memory queue to store traces, data compression with gzip, and queue configuration options (#4395)
+  - Make `Hclog2ZapLogger` struct and methods private for public go api review (#4431)
+- `newrelic` exporter (#4392):
+  - Marked unsupported metric as permanent error
+  - Force the interval to be valid even if 0
+- `awsxray` exporter: Add PHP stacktrace parsing support (#4454)
+- `file_storage` extension: Implementation of batch storage API (#4145)
+- `alibabacloudlogservice` exporter: Update labels to attributes for OTLP 0.9.0 (#4588)
+- `dynatrace` exporter: Update labels to attributes for OTLP 0.9.0 (#4590)
```

Comment:
let's remove all the _labels to attributes_ changes. those are internal changes not affecting end user